test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -138,5 +137,26 @@ Describe "Process Parent property" -Tags "CI" {
         $powershellexe = (Get-Process -Id $PID).mainmodule.filename
         & $powershellexe -noprofile -command '(Get-Process -Id $PID).Parent.Id' | Should -Be $PID
     }
+
+    It "Can find parent with spaces and parenthesis in the name on non-Windows" -Skip:($IsWindows) {
+        # Bug. See https://github.com/PowerShell/PowerShell/issues/12908
+        $commandName = 't ( e ( s ) t )'
+
+        $script = @'
+#!/bin/sh
+
+while true; do sleep 1; done
+'@
+
+        # Can't use testdrive: as unelevated user doesn't have perms in test in CI
```

Comment:
Please reword:
```suggestion
        # Can't use 'Testdrive:' as unelevated user doesn't have permissions to do 'chmod'.
```

I guess it is because .Net 7 RC1 has a breaking change in Directory.Create() where chmod mask was changed. If so, it might still break something in PowerShell or Pester.
/cc @nohwnd

Reply:
I assumed this was just a byproduct of how PowerShell CI is set up. I've not looked into it but I believe the unelevated user may potentially be inheriting the same tmpdir as the elevated account and thus cannot set the execute bit on files in that dir. This is just a guess so your assumption could also be true.